lib/comment.js

```diff
@@ -148,7 +147,7 @@ Comment.prototype['delete'] = function (query, fn) {
  * @api public
  */
 
-Comment.prototype.like = function() {
+Comment.prototype.like = function () {
```

Comment:
@retrofox technically because it's an anonymous function. practically because we are doing it everywhere else like that.

the only exception is that here we don't have any parameters to the function, which I could see as a valid reason not to add the space, but I'm not sure `eslint` is going to be very easy to configure that way. we could still strip the space and add an `ignore` comment, but I feel like soon this won't be a big deal if we adopt a coding standard such as Calypso's